nestjs/src/config/config.service.ts

```diff
@@ -35,12 +35,16 @@ type AWSServices = {
   accessKeyId: string;
 };
 
+const DEV_HOST = 'http://localhost:3000';
+const PROD_HOST = 'https://app.rs.school';
```

Comment:
Isn't it better to take them from the env file?

Reply:
i think current way is not what we meant: the host itself shouldn't be in the code. it should be in .env, ie the url itself should be set in the env variable, not boolean value